File: Easy Challenges/Challenge #0008 [Easy]/solutions/solution.py
#Python 3.X solution for Easy Challenge #0008
#GitHub: https://github.com/Ashkore
#https://www.reddit.com/user/Ashkoree/

string = " bottles of beer on the wall, take one down, pass it around."
bottles = 99
for x in range(bottles+1):
    print (str(bottles-x)+string)

#Extra Credit ??
newstring = ""
for x in range(bottles+1):
    newstring += str(bottles-x)+string
print (newstring)

File: Easy Challenges/Challenge #0008 [Easy]/solutions/solution.js
function main() {
  for (var i = 99; i >= 0; i--) {
    if (i > 1) {
      console.log(i + ' bottles of beer on the wall, ' + i + ' bottles of beer.');
      console.log('Take one down and pass it around, ' + (i - 1) + ' bottles of beer on the wall.');
    } else if (i == 1) {
      console.log(i + ' bottle of beer on the wall, ' + i + ' bottle of beer.');
      console.log('Take one down and pass it around, no more bottle of beer on the wall.');
    } else {
      console.log('No more bottles of beer on the wall, no more bottles of beer.');
      console.log('Go to the store and buy some more, 99 bottles of beer on the wall.');
    }
  }
}
main();
